### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by spigelli** [yuval-a/derivejs#3]
Any plans to add tests?

**Comment by yuval-a** [comment]
> Any plans to add tests?

Yes! One of the main things on the roadmap, and I'll really appreciate if someone can contribute and write them...
